dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -175,15 +175,15 @@ public String rssFeed() {
                 Workflow workflow = (Workflow)dbEntry;
                 Optional<WorkflowVersion> max = workflow.getWorkflowVersions().stream().filter(v -> v.getDbUpdateDate() != null)
                     .max(Comparator.comparing(Version::getDbUpdateDate));
-                entry.setTitle(workflow.getWorkflowPath() + (max.map(workflowVersion -> ":" + workflowVersion.getName()).orElse("")));
+                entry.setTitle(workflow.getWorkflowPath());
                 String workflowURL = createWorkflowURL(workflow);
                 entry.setGuid(workflowURL);
                 entry.setLink(workflowURL);
             } else if (dbEntry instanceof Tool) {
                 Tool tool = (Tool)dbEntry;
                 Optional<Tag> max = tool.getTags().stream().filter(v -> v.getDbUpdateDate() != null)
                     .max(Comparator.comparing(Version::getDbUpdateDate));
-                entry.setTitle(tool.getPath() + (max.map(tag -> ":" + tag.getName()).orElse("")));
+                entry.setTitle(tool.getPath());
                 String toolURL = createToolURL(tool);
                 entry.setGuid(toolURL);
```

Comment:
Experiment with an RSS reader and modify the description. 
User needs some indication of why they may be getting a new article (as sorted by v.getDbUpdateDate())

Reply:
Like a note saying 'sorted by most recent version'?

Reply:
Not exactly, its a bit more clear with a RSS reader. 
You get a new article when a new entry shows up and you should know from the article why you're getting it ('sorted by most recent version' is much too generic)
Solomon uses Thunderbird, I've tried Feedly (but haven't tried with Dockstore), there are also the alerts in #dockstore-alerts